cmd/stack.go

```diff
@@ -202,3 +204,9 @@ func validateServicesFlag(services []string) error {
 	}
 	return nil
 }
+
+func stripServicesWhitespace(services []string) {
```

Comment:
Nothing about this helper function is specific to services so you could probably name this more generically, like `trimStringSliceSpace` or something. And the change the argument name to `strs` or something.

Reply:
And then once you've done that, we should probably call this function from anywhere we also call `cmd.Flags().GetStringSlice(...)` in the codebase. So I'd suggest moving this function definition to `internal/common/helpers.go`. You'll need to rename it to start with an uppercase letter so it's exported. Then you can call it from here and other places via `common.TrimStringSliceSpace(...)`. Finally, you'll need to add an import at the top of this file's `import` section for `"github.com/elastic/elastic-package/internal/common"`.

Reply:
ok, will revise, thanks for feedback!

Reply:
@ycombinator is there _already_ a file named `helpers.go` that I should be using or am I making it?

Reply:
Sorry, you're making it.

Reply:
Ok have a look at https://github.com/elastic/elastic-package/pull/236/commits/70a2fa4b8c2fa0f1d5f08f432ac26173fa4b40a6.